mon-pix/app/pods/components/module/grain/style.scss

```
@@ -4,6 +4,13 @@
   border: 1px solid $pix-neutral-22;
   border-radius: $pix-spacing-s;
 
+  &__header {
+    margin: $pix-spacing-s;
+    padding: $pix-spacing-xs;
```

Je pense ici il devrait: 

```suggestion
    padding: $pix-spacing-xs $pix-spacing-m;
```